database/database.js

```diff
@@ -0,0 +1,27 @@
+const pgp = require('pg-promise')()
+const fs = require('fs')
+
+if(fs.existsSync('.env')) {
+  require('dotenv').config()
+}
+const connectionString = process.env.DATABASE_URL;
+const db = pgp(connectionString)
+
+const Twitter = {
+  getAllTweets: () => {
+    const psql = `SELECT * FROM twitbot`
```

Comment:
The variable name `psql` is too generic, I would assign it to a variable that is meaningfully named.
In this case consider something like `GET_ALL` the underscore and capital letters are a convention for static global variables so I think it would be appropriate for this query.